app/Models/Room.php

```diff
@@ -20,12 +20,19 @@ class Room extends Model
 
     protected $withCount = ['users'];
 
+    public static $roomColors = [
+        3 => '#76ff03', //light-green accent-3
+        2 => '#ffee58', //yellow lighten-2
+        1 => '#ff9100', //orange accent-2
+        0 => '#e0e0e0', //grey lighten-2
```

Comment:
I meant you do not need to use the # values. Pass 'light-green accent-3' as a css class.

Reply:
And you do not need this attribute. Just return them in the color() function. Or do you use this elsewhere?

Reply:
I generate the legend using the attribute. And as I wrote, the color classes set the `background-color` style property and in svg I need to change the `fill `property.